Rakefile

```
@@ -1,37 +1,66 @@
-
 sfe_root = ENV['SFE_ROOT']
+extension_name = ENV['SFE_NAME']
 
 unless sfe_root
   abort "SFE_ROOT not set"
-end
+else
+  extensions_dir = File.join(sfe_root, "extensions")
+  extension_dir = File.join(extensions_dir, extension_name)
+  install_lib_dir = File.join(extensions_dir, "__lib__")
 
-desc "Compile, build and install jar"
-task :default => "rubyfox"
+  unless File.directory?(install_lib_dir)
+    FileUtils.mkdir_p(install_lib_dir)
+  end
+end
 
 desc "Compile, build and install jar"
-task :rubyfox => "rubyfox:install"
+task :default => "rubyfox:install"
 
 namespace :rubyfox do
   desc "Compile, build and install jar"
-  task :install => [ :install_jruby, :install_jar ]
+  task :install => [:jar, :install_jruby, :install_jar, :copy_example]
 
   desc "Compile rubyfox"
   task :compile do
-    system "javac -d out -cp 'src/java:lib/*:#{sfe_root}/../../lib/*' src/java/com/neopoly/rubyfox/*.java"
+    puts "Compiling..."
+    if Gem.win_platform?
+      system "javac -d out -cp src/java;lib/*;#{sfe_root}/lib/* src/java/com/neopoly/rubyfox/*.java"
+    else
+      system "javac -d out -cp 'src/java:lib/*:#{sfe_root}/lib/*' src/java/com/neopoly/rubyfox/*.java"
+    end
   end
 
   desc "Build rubyfox.jar"
   task :jar => :compile do
+    puts "Bundling..."
     system 'jar cvMf target/rubyfox.jar -C out .'
   end
 
-  desc "Install jars to #{sfe_root}/../__lib__"
+  desc "Install jars to #{install_lib_dir}"
   task :install_jruby do
-    cp FileList["lib/*.jar"], "#{sfe_root}/../__lib__"
+    puts "Installing Jruby..."
```

It's either `JRuby` or `jruby` :sweat_smile: 

```suggestion
    puts "Installing JRuby..."
```